README.md

```diff
@@ -1,41 +1,112 @@
-# Zero Forcing Number
-This program calculates the zero forcing number (and set) of any graph. The zero forcing number is defined as follows:
->A subset S of initially infected vertices of a graph G is called forcing if we can infect the entire graph by iteratively applying the following process. At each step, any infected vertex which has a unique uninfected neighbour, infects this neighbour. The forcing number of G is the minimum cardinality of a forcing set in G.
+# zftools
 
-\- [Kalinowski, Kamčev, Sudakov](https://arxiv.org/abs/1705.10391)
+### Purpose.
+
+This software provides very fast calculation of zero forcing parameters.  Currently, it is able to compute only the “original” zero forcing parameter, namely the **zero forcing number of a finite simple graph**, originally defined by the “Special Graphs Working Group” that formed during a 2006 workshop at the American Institute of Mathematics (AIM) in their paper:
+
+<ul>
+AIM Minimum Rank – Special Graphs Work Group. Zero forcing sets and the minimum rank of graphs. <i>Linear Algebra and its Applications</i>, 428(7):1628–1648 (2008). (<a href="https://doi.org/10.1016/j.laa.2007.10.009">https://doi.org/10.1016/j.laa.2007.10.009</a>)
```

Comment:
Can we make the link a bit cleaner? Perhaps something like:

- AIM Minimum Rank – Special Graphs Work Group. [Zero forcing sets and the minimum rank of graphs](https://doi.org/10.1016/j.laa.2007.10.009). <i>Linear Algebra and its Applications</i>, 428(7):1628–1648 (2008).

Reply:
I think that might be a MLA citation or something though, so maybe not. Just an idea.

Reply:
The DOI is a sort of universal identifier, but still I like the idea of just making it a hyperlink from the name of the paper instead of showing the whole ugly URL there.  I will make this change!